assets/i18n/en.json

```diff
@@ -283,12 +287,14 @@
     "I18N_LEARNER_DASHBOARD_EMPTY_COMPLETED_COLLECTIONS": "It looks like you haven't completed any collections yet. Head over to the library to start an exciting new collection!",
     "I18N_LEARNER_DASHBOARD_EMPTY_COMPLETED_EXPLORATIONS": "It looks like you haven't completed any explorations yet. Head over to the library to start an exciting new exploration!",
     "I18N_LEARNER_DASHBOARD_EMPTY_COMPLETED_GOALS_SECTION": "Complete a Goal from above and see your progress here when it is completed!",
+    "I18N_LEARNER_DASHBOARD_EMPTY_COMPLETED_STORIES_SECTION": "Hmm this looks empty... Head over to the <a href=\"http://localhost:8181/learn/math\" class=\"inline-links\">classroom</a> to complete an exciting new story!",
```

Comment:
Ditto here and elsewhere my comment in your previous PR on links.

Reply:
Actually, this seems like the change from the same commit, in which case, ignore this and you can just wait for the other PR to be merged.

Reply:
Done as per discussion!